clang/test/CodeGenSYCL/kernel_functor.cpp

```diff
@@ -1,5 +1,7 @@
-// RUN: %clang_cc1 -fsycl-is-device -fsycl-int-header=%t.h %s -o %t.spv
-// RUN: FileCheck %s --input-file=%t.h
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -fsycl-is-device -fno-sycl-unnamed-lambda -fsycl-int-header=%t.h %s -o %t.spv
+// RUN: FileCheck %s --input-file=%t.h --check-prefixes=NUL,CHECK
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -fsycl-is-device -fsycl-int-header=%t.h %s -o %t.spv
```

Comment:
Is `%t.spv` intended to be unused output?

Reply:
Yes, we only care about the int-header, but have to specify a -o anyway.  Presumably these could all just do a -o -, but I think that messes up the llvm-lit output.

Reply:
I think we usually do `-o /dev/null` so that we don't pay the cost of writing out to a temp file (and deleting it) that we don't care about.

Reply:
Does that work on windows?!